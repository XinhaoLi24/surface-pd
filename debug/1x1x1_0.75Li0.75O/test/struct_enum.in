Na0.75 Li2 Ni4 O7 F0.75
bulk
2.774107 0.000000 0.115671
1.384605 2.568215 0.116551
0.000000 0.000000 33.323051
3
2
2.943958 0.987070 13.112058 0/1
2.800358 1.162875 16.555934 1/2
4 4
0.001
full
30 30 80
20 20 80
30 30 80
