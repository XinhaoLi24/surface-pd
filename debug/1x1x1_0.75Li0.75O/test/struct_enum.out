 # <fortpy version="3" revision="247"></fortpy>
Na0.75 Li2
bulk
  2.7741070       0.0000000      0.11567100        # a1 parent lattice vector
  1.3846050       2.5682150      0.11655100        # a2 parent lattice vector
  0.0000000       0.0000000       33.323051        # a3 parent lattice vector
    2 # Number of points in the multilattice
  2.9439580      0.98707000       13.112058        # d01 d-vector, labels: 0/1
  2.8003580       1.1628750       16.555934        # d02 d-vector, labels: 1/2
 3-nary case
   4   4 # Starting and ending cell sizes for search
 0.10000000E-02 # Epsilon (finite precision parameter)
Concentration check:
    T
Including only structures of which the concentration   of each atom is in the range:
Type 1:    3/   8 --    3/   8
Type 2:    1/   4 --    1/   4
Type 3:    3/   8 --    3/   8
full list of labelings (including incomplete labelings) is used
(Non)Equivalency list: 1  2
start   #tot      HNF     Hdegn   labdegn   Totdegn   #size idx    pg    SNF             HNF                 Left transform                          labeling
          1         1       1        8        8           1   4     4   1  1  4    1  0  1  0  0  4      1    0    0    0    1    0    0    0    1   00011222
          2         2       1        8        8           2   4     4   1  1  4    1  0  1  0  1  4      1    0    0    0    1    0    0   -1    1   00011222
          3         3       1        8        8           3   4     4   1  1  4    1  0  1  0  2  4      1    0    0    0    1    0    0   -2    1   00011222
          4         4       1        8        8           4   4     4   1  1  4    1  0  1  0  3  4      1    0    0    0    1    0    0   -3    1   00011222
          5         5       1        8        8           5   4     4   1  1  4    1  0  2  0  1  2      1    0    0    0    0    1    0   -1    2   00011222
          6         6       2        8       16           6   4     4   1  1  4    1  0  4  0  0  1      1    0    0    0    0    1    0    1    0   00011222
          7         1       1        8        8           7   4     4   1  1  4    1  0  1  0  0  4      1    0    0    0    1    0    0    0    1   00012122
          8         2       1        8        8           8   4     4   1  1  4    1  0  1  0  1  4      1    0    0    0    1    0    0   -1    1   00012122
          9         3       1        8        8           9   4     4   1  1  4    1  0  1  0  2  4      1    0    0    0    1    0    0   -2    1   00012122
         10         4       1        8        8          10   4     4   1  1  4    1  0  1  0  3  4      1    0    0    0    1    0    0   -3    1   00012122
         11         5       1        8        8          11   4     4   1  1  4    1  0  2  0  1  2      1    0    0    0    0    1    0   -1    2   00012122
         12         6       2        8       16          12   4     4   1  1  4    1  0  4  0  0  1      1    0    0    0    0    1    0    1    0   00012122
         13         1       1        8        8          13   4     4   1  1  4    1  0  1  0  0  4      1    0    0    0    1    0    0    0    1   00012212
         14         2       1        8        8          14   4     4   1  1  4    1  0  1  0  1  4      1    0    0    0    1    0    0   -1    1   00012212
         15         3       1        8        8          15   4     4   1  1  4    1  0  1  0  2  4      1    0    0    0    1    0    0   -2    1   00012212
         16         4       1        8        8          16   4     4   1  1  4    1  0  1  0  3  4      1    0    0    0    1    0    0   -3    1   00012212
         17         5       1        8        8          17   4     4   1  1  4    1  0  2  0  1  2      1    0    0    0    0    1    0   -1    2   00012212
         18         6       2        8       16          18   4     4   1  1  4    1  0  4  0  0  1      1    0    0    0    0    1    0    1    0   00012212
         19         1       1        8        8          19   4     4   1  1  4    1  0  1  0  0  4      1    0    0    0    1    0    0    0    1   00012221
         20         2       1        8        8          20   4     4   1  1  4    1  0  1  0  1  4      1    0    0    0    1    0    0   -1    1   00012221
         21         3       1        8        8          21   4     4   1  1  4    1  0  1  0  2  4      1    0    0    0    1    0    0   -2    1   00012221
         22         4       1        8        8          22   4     4   1  1  4    1  0  1  0  3  4      1    0    0    0    1    0    0   -3    1   00012221
         23         5       1        8        8          23   4     4   1  1  4    1  0  2  0  1  2      1    0    0    0    0    1    0   -1    2   00012221
         24         6       2        8       16          24   4     4   1  1  4    1  0  4  0  0  1      1    0    0    0    0    1    0    1    0   00012221
         25         7       2        8       16          25   4     2   1  1  4    1  0  1  1  0  4      1    0    0    0    1    0   -1    0    1   00011222
         26         8       2        8       16          26   4     2   1  1  4    1  0  1  1  1  4      1    0    0    0    1    0   -1   -1    1   00011222
         27         9       2        8       16          27   4     2   1  1  4    1  0  1  1  2  4      1    0    0    0    1    0   -1   -2    1   00011222
         28        10       2        8       16          28   4     2   1  1  4    1  0  1  1  3  4      1    0    0    0    1    0   -1   -3    1   00011222
         29         7       2        8       16          29   4     2   1  1  4    1  0  1  1  0  4      1    0    0    0    1    0   -1    0    1   00012122
         30         8       2        8       16          30   4     2   1  1  4    1  0  1  1  1  4      1    0    0    0    1    0   -1   -1    1   00012122
         31         9       2        8       16          31   4     2   1  1  4    1  0  1  1  2  4      1    0    0    0    1    0   -1   -2    1   00012122
         32        10       2        8       16          32   4     2   1  1  4    1  0  1  1  3  4      1    0    0    0    1    0   -1   -3    1   00012122
         33         7       2        8       16          33   4     2   1  1  4    1  0  1  1  0  4      1    0    0    0    1    0   -1    0    1   00012212
         34         8       2        8       16          34   4     2   1  1  4    1  0  1  1  1  4      1    0    0    0    1    0   -1   -1    1   00012212
         35         9       2        8       16          35   4     2   1  1  4    1  0  1  1  2  4      1    0    0    0    1    0   -1   -2    1   00012212
         36        10       2        8       16          36   4     2   1  1  4    1  0  1  1  3  4      1    0    0    0    1    0   -1   -3    1   00012212
         37         7       2        8       16          37   4     2   1  1  4    1  0  1  1  0  4      1    0    0    0    1    0   -1    0    1   00012221
         38         8       2        8       16          38   4     2   1  1  4    1  0  1  1  1  4      1    0    0    0    1    0   -1   -1    1   00012221
         39         9       2        8       16          39   4     2   1  1  4    1  0  1  1  2  4      1    0    0    0    1    0   -1   -2    1   00012221
         40        10       2        8       16          40   4     2   1  1  4    1  0  1  1  3  4      1    0    0    0    1    0   -1   -3    1   00012221
         41        11       2        8       16          41   4     2   1  1  4    1  0  1  2  0  4      1    0    0    0    1    0   -2    0    1   00011222
         42        12       1        8        8          42   4     2   1  1  4    1  0  2  1  1  2      1    0    0   -1    0    1   -2   -1    2   00011222
         43        11       2        8       16          43   4     2   1  1  4    1  0  1  2  0  4      1    0    0    0    1    0   -2    0    1   00012122
         44        12       1        8        8          44   4     2   1  1  4    1  0  2  1  1  2      1    0    0   -1    0    1   -2   -1    2   00012122
         45        11       2        8       16          45   4     2   1  1  4    1  0  1  2  0  4      1    0    0    0    1    0   -2    0    1   00012212
         46        12       1        8        8          46   4     2   1  1  4    1  0  2  1  1  2      1    0    0   -1    0    1   -2   -1    2   00012212
         47        11       2        8       16          47   4     2   1  1  4    1  0  1  2  0  4      1    0    0    0    1    0   -2    0    1   00012221
         48        12       1        8        8          48   4     2   1  1  4    1  0  2  1  1  2      1    0    0   -1    0    1   -2   -1    2   00012221
         49        13       1       16       16          49   4     4   1  1  4    1  0  1  2  1  4      1    0    0    0    1    0   -2   -1    1   00011222
         50        14       1       16       16          50   4     4   1  1  4    1  0  1  2  3  4      1    0    0    0    1    0   -2   -3    1   00011222
         51        13       1        8        8          51   4     4   1  1  4    1  0  1  2  1  4      1    0    0    0    1    0   -2   -1    1   00012122
         52        14       1        8        8          52   4     4   1  1  4    1  0  1  2  3  4      1    0    0    0    1    0   -2   -3    1   00012122
         53        13       1        8        8          53   4     4   1  1  4    1  0  1  2  1  4      1    0    0    0    1    0   -2   -1    1   00012221
         54        14       1        8        8          54   4     4   1  1  4    1  0  1  2  3  4      1    0    0    0    1    0   -2   -3    1   00012221
         55        15       2        8       16          55   4     2   1  1  4    1  0  1  3  0  4      1    0    0    0    1    0   -3    0    1   00011222
         56        16       2        8       16          56   4     2   1  1  4    1  0  1  3  1  4      1    0    0    0    1    0   -3   -1    1   00011222
         57        17       2        8       16          57   4     2   1  1  4    1  0  1  3  2  4      1    0    0    0    1    0   -3   -2    1   00011222
         58        18       2        8       16          58   4     2   1  1  4    1  0  1  3  3  4      1    0    0    0    1    0   -3   -3    1   00011222
         59        15       2        8       16          59   4     2   1  1  4    1  0  1  3  0  4      1    0    0    0    1    0   -3    0    1   00012122
         60        16       2        8       16          60   4     2   1  1  4    1  0  1  3  1  4      1    0    0    0    1    0   -3   -1    1   00012122
         61        17       2        8       16          61   4     2   1  1  4    1  0  1  3  2  4      1    0    0    0    1    0   -3   -2    1   00012122
         62        18       2        8       16          62   4     2   1  1  4    1  0  1  3  3  4      1    0    0    0    1    0   -3   -3    1   00012122
         63        15       2        8       16          63   4     2   1  1  4    1  0  1  3  0  4      1    0    0    0    1    0   -3    0    1   00012212
         64        16       2        8       16          64   4     2   1  1  4    1  0  1  3  1  4      1    0    0    0    1    0   -3   -1    1   00012212
         65        17       2        8       16          65   4     2   1  1  4    1  0  1  3  2  4      1    0    0    0    1    0   -3   -2    1   00012212
         66        18       2        8       16          66   4     2   1  1  4    1  0  1  3  3  4      1    0    0    0    1    0   -3   -3    1   00012212
         67        15       2        8       16          67   4     2   1  1  4    1  0  1  3  0  4      1    0    0    0    1    0   -3    0    1   00012221
         68        16       2        8       16          68   4     2   1  1  4    1  0  1  3  1  4      1    0    0    0    1    0   -3   -1    1   00012221
         69        17       2        8       16          69   4     2   1  1  4    1  0  1  3  2  4      1    0    0    0    1    0   -3   -2    1   00012221
         70        18       2        8       16          70   4     2   1  1  4    1  0  1  3  3  4      1    0    0    0    1    0   -3   -3    1   00012221
         71        19       2        8       16          71   4     4   1  2  2    1  0  2  0  0  2      1    0    0    0    1    0    0    0    1   00011222
         72        19       2        8       16          72   4     4   1  2  2    1  0  2  0  0  2      1    0    0    0    1    0    0    0    1   00012122
         73        19       2        8       16          73   4     4   1  2  2    1  0  2  0  0  2      1    0    0    0    1    0    0    0    1   00012212
         74        19       2        8       16          74   4     4   1  2  2    1  0  2  0  0  2      1    0    0    0    1    0    0    0    1   00012221
         75        20       2        8       16          75   4     2   1  2  2    1  0  2  1  0  2      1    0    0    0    1    0   -1    0    1   00011222
         76        20       2        8       16          76   4     2   1  2  2    1  0  2  1  0  2      1    0    0    0    1    0   -1    0    1   00012122
         77        20       2        8       16          77   4     2   1  2  2    1  0  2  1  0  2      1    0    0    0    1    0   -1    0    1   00012212
         78        20       2        8       16          78   4     2   1  2  2    1  0  2  1  0  2      1    0    0    0    1    0   -1    0    1   00012221
         79        21       1        8        8          79   4     2   1  2  2    1  1  2  0  0  2      1    0    0   -1    1    0    0    0    1   00011222
         80        21       1        8        8          80   4     2   1  2  2    1  1  2  0  0  2      1    0    0   -1    1    0    0    0    1   00012122
         81        21       1        8        8          81   4     2   1  2  2    1  1  2  0  0  2      1    0    0   -1    1    0    0    0    1   00012212
         82        21       1        8        8          82   4     2   1  2  2    1  1  2  0  0  2      1    0    0   -1    1    0    0    0    1   00012221
         83        22       1        8        8          83   4     4   1  2  2    1  1  2  1  0  2      1    0    0   -1    1    0   -1    0    1   00011222
         84        22       1       16       16          84   4     4   1  2  2    1  1  2  1  0  2      1    0    0   -1    1    0   -1    0    1   00012122
         85        22       1        8        8          85   4     4   1  2  2    1  1  2  1  0  2      1    0    0   -1    1    0   -1    0    1   00012221
         86        23       1       16       16          86   4     4   1  2  2    2  0  1  0  1  2      0    1    0    1    0    0    0   -1    1   00011222
         87        23       1        8        8          87   4     4   1  2  2    2  0  1  0  1  2      0    1    0    1    0    0    0   -1    1   00012122
         88        23       1        8        8          88   4     4   1  2  2    2  0  1  0  1  2      0    1    0    1    0    0    0   -1    1   00012221
